aws/cloudformation/components/ami.yml.erb

```diff
@@ -65,7 +65,7 @@ frontend_device_name ||= '/dev/sda1'
       InstanceType: <%=instance_type%>
       IamInstanceProfile: !Ref FrontendInstanceProfile
       SecurityGroupIds: [!ImportValue VPC-FrontendSecurityGroup]
-      SubnetId: !ImportValue VPC-Subnet<%=azs.first%>
+      SubnetId: <%=subnets.first.to_json%>
```

Comment:
Just curious, how does it know here to use the local variable intead of the method from `vpc.rb`?

Reply:
I think this just follows the normal Ruby scoping, where this local variable takes precedence over the method of the same name.